.gitignore

```diff
@@ -360,3 +360,5 @@ MigrationBackup/
 # venv
 env/
 website/docusaurus/
+
+tools/awps-tunnel/client/public/examples/
```

Comment:
put into `tools/awps-tunnel/client/public/spec`, `tools/awps-tunnel/client/public/spec/{version}/webpubsub.json` and `.../{version}/examples/...`